azdev/utilities/venv.py

```diff
@@ -0,0 +1,153 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+import os
+import subprocess
+import platform
+from knack.util import CLIError
+import azdev.operations.extensions
+from azdev.utilities import display, shell_cmd
+from . import const
+
+
+def edit_activate(azure_config_path, dot_azure_config, dot_azdev_config):
+    if const.IS_WINDOWS:
+        ps1_edit(azure_config_path, dot_azure_config, dot_azdev_config)
+        bat_edit(azure_config_path, dot_azure_config, dot_azdev_config)
+    else:
+        unix_edit(azure_config_path, dot_azure_config, dot_azdev_config)
+
+
+def edit_pyenv_activate(activate_path):
+    set_azdev_content = 'set -gx AZDEV_CONFIG_DIR "${prefix}/.azdev";\n'
```

Comment:
These should probably be constants in the const file

Reply:
No need. 
1. they are not global constants used across files/methods.
2. as @haroldrandom  commented (https://github.com/Azure/azure-cli-dev-tools/pull/264#discussion_r513373912), we will just put very frequently used constants to const.py.